tests/instruments/contributors.enaml

```diff
@@ -104,8 +104,7 @@ enamldef InstrContributor1(PluginManifest):
             id = 'false_connection'
             description = 'False connection 1'
             new => (workbench, defaults, read_only):
-                return TestConnection(declaration=self,
-                                      defaults=defaults,
+                return TestConnection(defaults=defaults,
```

Comment:
To reduce the complexity in writing new connections the declaration is set by the plugin when creating the connection, which guarantees that it is not None even if the user did not set it. The test ensures it is so.